src/app/components/FormElements/DateTimeField.tsx

```diff
@@ -4,16 +4,12 @@ import { default as FormFieldWrapper, FormFieldWrapperProps } from './FormFieldW
 import * as DateTime from 'react-datetime';
 import 'react-datetime/css/react-datetime.css';
 
-export interface DateTimeFieldProps {
-    defaultValue?: string;
-}
-
-export default class DateTimeield extends React.PureComponent<FormFieldWrapperProps & DateTimeFieldProps>{
+export default class DateTimeField extends React.PureComponent<FormFieldWrapperProps>{
     render(){
-        const {input:{value, onChange}, defaultValue=moment().startOf('day').add(9, 'hours')} = this.props;
```

Comment:
We just need to figure out how to pass properties into these fields.  It looks like we should be able to use: https://redux-form.com/7.2.3/docs/api/field.md/#2-a-stateless-function

I tried this in code and typescript seems to be cool with it:

```ts
<Field name="startTime" component={(f)=><DateTimeField {...f}/>} label="Start Time" validate={[Validators.required]}/>
```
This example only required that we change the label property to be **nullable**

The props that are passed (in the example above in the variable **`f`**) are described here: 
https://redux-form.com/7.2.3/docs/api/field.md/#props

Note that it says that any custom properties passed into the Field will be passed down to the component and merged in with the input and meta properties